src/pages/home/report/FloatingMessageCounter/index.js

```diff
@@ -25,7 +25,7 @@ const defaultProps = {
     onClick: () => {},
 };
 
-const MARKER_INACTIVE_TRANSLATE_Y = -30;
+const MARKER_INACTIVE_TRANSLATE_Y = -40;
```

Comment:
Why did we change this value?

Reply:
To completely hide the `FloatingMessageCounter` component when we are using a larger font size.
![Simulator Screen Shot - iPhone 13 - 2022-12-01 at 07 46 16](https://user-images.githubusercontent.com/51829206/204976224-db486e18-2f09-4511-9bcc-b6d74fe1fb81.png)